react-nd/src/Routes.jsx

```diff
@@ -0,0 +1,44 @@
+import { RouterProvider, createBrowserRouter } from "react-router-dom";
+
+import DefaultLayout from '@/layout/DefaultLayout';
+import ErrorPage from '@/views/ErrorPage';
+import Home from '@/views/home/Home';
+import Category from '@/views/category/Category';
+import About from '@/views/about/About';
+import Login from '@/views/login/Login';
+  
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <DefaultLayout />,
+    errorElement: <ErrorPage />,
+    children: [
+      {
+        path: "/",

```

Comment:
url kelius geriausia būtų laikyti atskirame objekta ir užnaudoti konfiguruojant routus ir atvaizduojant <Link to={routes.home}, tokiu būdu sumažinsite klaidų tikimybę ir galėsite lengviau keisti kelius ateityje